.travis.yml

```diff
@@ -4,11 +4,10 @@ android:
   components:
     - tools
     - platform-tools
-    - build-tools-23.0.2
-    - android-23
+    - build-tools-25.0.2
+    - android-25
 
 jdk:
-  - oraclejdk7
```

Comment:
yep.  i experimented with having a conditional block in case for JDK7 and JDK8 travis builds, but removing it altogether works for me.  @pyricau any thoughts for keeping the JDK7 build around?